### PR TITLE
Add script for generating client short tokens

### PR DESCRIPTION
**Pull request by jonathangreen**
## Description

Add a script that can generate long lived client short tokens. We can use this to generate tokens for Adobe to test with that won't expire as quickly.

## Motivation and Context

This will hopefully simplify communication with Adobe and get our registry approved faster.

## Testing

Should be able to do something like this to generate short tokens:
```
$ ./generate_short_token --id adb-vend-id-2 --days 30 centaur-adobe-test
```

### CODE REVIEW

**Review by tdilauro — COMMENTED** [review]
I think it's worth getting this into the code base. A couple of suggestions:
- Move the `ShortTokenScript` class into `scripts.py`.
- Specify the token duration in hours, rather than days (or allow both), which would better would be more useful/appropriate for my use cases.
- Move the script into either `./bin/util` or `./bin`.

File: scripts.py [hunk 1]
```diff
@@ -1880,3 +1883,77 @@ def do_run(self, output=sys.stdout, cmd_args=None, exporter=None):
 
         exporter = exporter or LocalAnalyticsExporter()
         output.write(exporter.export(self._db, start, end))
+
+class GenerateShortTokenScript(LibraryInputScript):
+    """
+    Generate long lived client short tokens that can be used for testing the service. This
```
Comment (tdilauro) [inline scripts.py:1889]: Minor: This script can generate tokens of widely varying lengths (1 minute and up). I'd suggest the following wording for the first sentence: "Generate a short client token of the specified duration that can be used for testing that involves the Adobe Vendor ID API implementation."

File: scripts.py [hunk 2]
```diff
@@ -1880,3 +1883,77 @@ def do_run(self, output=sys.stdout, cmd_args=None, exporter=None):
...
+        parser.add_argument(
+            '--barcode',
+            help="The patron barcode.",
+            required=True,
+        )
+        parser.add_argument(
+            '--pin',
+            help="The patron pin."
+        )
+        group = parser.add_mutually_exclusive_group(required=True)
```
[the remark this answers is not in the document]
Comment (jonathangreen) [reply scripts.py:1906]: I didn't think of that! Like if you wanted to make the token last for `--hours 10 --minutes 5`.

[the remark this answers is not in the document]
Comment (tdilauro) [reply scripts.py:1905]: Yeah, exactly.

**Comment by tdilauro** [comment]
This looks good to go, assuming all tests pass.

**Pull request merged**